src/components/Button/index.examples.js

```diff
@@ -8,14 +8,14 @@ const standardButtonExamples = [
   {type: 'primary', children: 'Primary'},
   {type: 'success', children: 'Success'},
   {type: 'warning', children: 'Warning'},
-  {type: 'danger', children: 'Danger'}
+  {type: 'danger', children: 'Danger'},
+  {type: 'orange', children: 'Orange'},
+  {type: 'turquoise', children: 'turquoise'}
```

Comment:
My general concern is having an insane skittles bag of colors and special vocabulary. 

`<Button small red pill outline>`

Is way nicer to me than some word that has special meaning. Semantically this could be wrapped for specific use cases. 

I still hate the "type" property as string based and generally "meh"